packages/block-editor/src/components/iframe/index.js

```diff
@@ -254,6 +254,14 @@ function Iframe( {
 				height: auto !important;
 				min-height: 100%;
 			}
+			/*
+				This CSS Custom Properties aren't used anymore as defaults, but we still need to keep them for backward compatibility.
+				See: https://github.com/WordPress/gutenberg/pull/37381
+			 */
+			:root {
```

Comment:
I could see this being abstracted into a filter or function so we could house any backwards compat CSS properties there. 

For now, it's a quick fix.